plane/source/docs/tuning-cruise.rst

```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
 
-When using an airspeed sensor in FBWB or CRUISE, the target airspeed can be directly controlled with the throttle stick position. Mid throttle will set the speed as halfway between :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` (high stick) and :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` (low stick). Using :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` bit 3 you can force the target airspeed in these modes to always be :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` ignoring throttle position. Alternatively, Bit 1 of :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` can be set to make the center throttle stick position be :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` instead of midway between :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` (high stick) and :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` (low stick).
+When Airspeed Sensor Equipped
+-----------------------------
+
+In FBWB or CRUISE, the target airspeed can be directly controlled with the throttle stick position. Mid throttle will set the speed as halfway between :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` (high stick) and :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` (low stick). Using :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` bit 3 you can force the target airspeed in these modes to always be :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` ignoring throttle position. Alternatively, Bit 1 of :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` can be set to make the center throttle stick position be :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` instead of midway between :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` (high stick) and :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` (low stick).
+
+.. note:: The :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` parameter should be adjusted to the average throttle value used at cruise speed. It optimizes the bias point for the speed control loops and is used in case of airspeed failure.
+
+While cruising, the artificial horizon in the OSD or GCS may show an average positive or negative pitch above the level indicator (ie fuselage/autopilot level). This means that the "trim level" pitch or AOA is different than what was set during the accelerometer calibration step. This can be trimmed out so that non altitude controlled modes fly at the same speed and throttle. Do this by adjusting the pitch trim, adding the desired degrees nose up or down using :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. This also optimizes the speed control loop bias point and will allow non altitude controlled modes to fly level at the same throttle and speed. 
+
+
+Without an Airspeed Sensor
+--------------------------
+
+Without an airspeed sensor, both the pitch trim and the :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` parameter would need to be changed appropriately for the desired mid-stick cruise speed. 
+
+.. tip:: Often small light planes or gliders need 2 or 3 degrees of pitch trim to fly at their optimum cruising speed/throttle rather than at the fuselage/autopilot level pitch. This can be done at setup by:
+
+- **(Preferred)** Add the desired degrees nose up(usually) or down to :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. 
```

Comment:
These bullet points are not exclusive to small aircraft. They are important for all aircraft.

Reply:
rewording a bit to allow it to apply to all